internal/guides/generated/producer/setters.go

```diff
@@ -51,25 +42,43 @@ in this guide.
 
 {{< svg src="images/set-command" >}}
 
+This guide walks you through an end-to-end example to create, invoke and delete setters.
+
+## Steps
+
+1. [Creating a Setter](#creating-a-setter)
+2. [Targeting fields using the path](#targeting-fields-using-the-path)
+3. [Invoking a Setter](#invoking-a-setter)
+4. [Deleting a Setter](#deleting-a-setter)
+5. [OpenAPI Validations](#openapi-validations)
+6. [Required Setters](#required-setters)
+7. [Setting Lists](#setting-lists)
+
 #### Creating a Setter
 
 Setters may be created either manually (by editing the Kptfile directly), or
-programmatically (through the ` + "`" + `create-setter` + "`" + ` command).  The ` + "`" + `create-setter` + "`" + `
+programmatically (through the ` + "`" + `create-setter` + "`" + ` command). The ` + "`" + `create-setter` + "`" + `
 command will:
 
 1. create a new OpenAPI definition for a setter in the Kptfile
 2. create references to the setter definition on the resource fields
 
+<!-- @createResource @verifyGuides-->
+
+  cat <<EOF >deployment.yaml
   # deployment.yaml -- original
   kind: Deployment
   metadata:
     name: foo
   spec:
     replicas: 3
+  EOF
+
+<!-- @createSetter @verifyGuides-->
```

Comment:
We should update the job that generates these guides for the cli to drop these html comments.